res/img/element-icons/brands/apple.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="25" height="24" viewBox="0 0 25 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
(Off-topic) I'm not really sure why we have an `element-icons` directory at all... But anyway, it's been around for a while.

Reply:
I think it was for when the rebrand replaced a bunch but not all of the icons